crates/bevy_diagnostic/src/diagnostic.rs

```diff
@@ -83,19 +97,38 @@ impl Diagnostic {
         self
     }
 
+    /// The smoothing factor used for the exponential smoothing used by
+    /// [`smoothed`](Self::smoothed).
+    ///
+    /// If measurements come in less frequently than `smoothing_factor` seconds
+    /// apart, no smoothing will be applied. As measurements come in more
+    /// frequently, the takes a greater effect such that it takes approximately
+    /// `smoothing_factor` seconds to have 83% of an instantaneous change in the
+    /// measurement to be reflected in the smoothed value.
+    ///
+    /// A value of 0 will effectively disable smoothing.
+    #[must_use]
+    pub fn with_smoothing_factor(mut self, smoothing_factor: f64) -> Self {
```

Comment:
should this function be setting self.ema_smoothing_factor? The above docs and the function below don't seem to match.

Reply:
Whoops, yes, you're correct.